expectations-of-foundation.md

```diff
@@ -0,0 +1,20 @@
+# Expectations
+With the intent to merge the Node.js Foundation and JS Foundation being announced, those interested in the continued support of Node.js and JS-related projects are working through expectations and structure for what the potential new foundation could look like. 
+
+This document is for the initial phase of work for the Bootstrap team to draft requirements that overlap with scope of the Board in order to provide an outline and structure for the foundation attuned to the needs of the projects and ecosystem stakeholders who have participated.
+
+- Communicated scope for the foundation and board
+- Annual reviews of staff, NPS, board, programs, support services to projects for accountability and incremental improvement
+- The foundation staff, executive director, and board should follow a "servant leadership" style.
+- Project representation on the Board
+  - Projects could have tiers, only 'platinum' projecs should sit on the Board directly. Other projects would have shared seats(like gold/silver member strcuture)
+  - Community representation on the Board: 1 seat guaranteed for the Cross-Project Community Council and 1 seat for Individual Membership program if and only if threshold requirements have been met(TODO)
+  - Each council(CPC and Community Council) should have a given allocation of Board seats. How these are allocated should be defined by each respective council.
+- Board must establish Cross Project Council focused on the specific needs of the member projects
+  - Allow projects to self-organize amongst themselves. Processes are defined and driven by project e.g. CPC is not structured by the Board
+  - CPC should be co-chaired by the Executive Director of the foundation
```

Comment:
This seems pre mature given that we haven't locked down governance yet.

we should likely drop this for now